opentelemetry-auto-instrumentation/src/opentelemetry/auto_instrumentation/bootstrap.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import pkgutil
+import subprocess
+import sys
+from logging import getLogger
+
+logger = getLogger(__file__)
+
+
+# target library to desired instrumentor path/versioned package name
+instrumentations = {
+    "dbapi": "opentelemetry-ext-dbapi>=0.8.8",
+    "django": "opentelemetry-ext-django>=0.8.8",
+    "flask": "opentelemetry-ext-flask>=0.8.8",
+    "grpc": "opentelemetry-ext-grpc>=0.8.8",
+    "requests": "opentelemetry-ext-requests>=0.8.8",
+    "jinja2": "opentelemetry-ext-jinja2>=0.8.8",
+    "mysql": "opentelemetry-ext-mysql>=0.8.8",
+    "psycopg2": "opentelemetry-ext-psycopg2>=0.8.8",
+    "pymongo": "opentelemetry-ext-pymongo>=0.8.8",
+    "pymysql": "opentelemetry-ext-pymysql>=0.8.8",
+    "redis": "opentelemetry-ext-redis>=0.8.8",
+    "sqlalchemy": "opentelemetry-ext-sqlalchemy>=0.8.8",
+    "wsgi": "opentelemetry-ext-wsgi>=0.8.8",
```

Comment:
Are the version numbers correct here? I would expect them to be `0.8b0`

Reply:
No idea how they crept in. I think search/replace command messed them up. Fixed now. Thanks.

Reply:
I'm still seeing 0.8.8 unless im missing something

Reply:
My bad. Should be fixed now. The push didn't go through last time.